libavogadro/src/extensions/yaehmop/CMakeLists.txt

```diff
@@ -0,0 +1,69 @@
+include_directories(${CMAKE_CURRENT_BINARY_DIR})
+
+set(LINK_LIBS avogadro)
+set(PLUGIN_LABEL extensions)
+set(PLUGIN_TARGET extensions)
+
+set(yaehmopextension_SRCS
+  yaehmopbanddialog.cpp
+  yaehmopextension.cpp
+  yaehmopout.cpp
+)
+
+set(yaehmopextension_UIS
+  yaehmopbanddialog.ui
+)
+
+avogadro_plugin_nogl(yaehmopextension
+  "${yaehmopextension_SRCS}";
+  "${yaehmopextension_UIS}"
+)
+
+# Download Yaehmop executable
+# Let's set the name. Windows likes to add '.exe' at the end
+if(WIN32)
+  set(YAEHMOP_NAME "yaehmop.exe")
+else(WIN32)
+  set(YAEHMOP_NAME "yaehmop")
+endif(WIN32)
+
+if(NOT EXISTS ${CMAKE_BINARY_DIR}/bin/${YAEHMOP_NAME})
+  # Linux
+  if(UNIX AND NOT APPLE)
+    set(YAEHMOP_DOWNLOAD_LOCATION "https://github.com/psavery/yaehmop/releases/download/3.0/linux64-yaehmop.tgz")
```

Comment:
I think we want to have a version number as a parameter, making it a bit easier to adapt to v. 3.1., etc.